### PR TITLE
Fix/patient flag code systems

### PR DESCRIPTION
**Pull request by vickyjaiswal0308**

### CODE REVIEW

File: codesystems/CodeSystem-England-ConditionCodeRA.xml [hunk 1]
```diff
@@ -29,50 +29,50 @@
...
   <count value="11" />
   <concept>
-    <code value="1" />
+    <code value="vision" />
     <display value="Vision (for example blindness or partial sight)" />
   </concept>
   <concept>
-    <code value="2" />
+    <code value="hearing" />
     <display value="Hearing (for example deafness or partial hearing)" />
```
Comment (ryma2fhir) [inline codesystems/CodeSystem-England-ConditionCodeRA.xml:37]: Should the codes not display the negative, eg. hearing impairment? Thoughts @marcusfearnett-nhsdigital ?

These display values also seem very long, it feels like this should be the [definition](https://hl7.org/fhir/R4/codesystem-definitions.html#CodeSystem.concept.definition), e.g.
><code value="2" />
    <code value="hearing-impairment" />
    <display value="Hearing Impairment" />
    <definition vaue = "hearing impairment, for example deafness or partial hearing." />

**Comment by vickyjaiswal0308** [comment]
> there's also an issue in the validator that needs fixing

It's fixed now

File: codesystems/CodeSystem-England-ConditionCodeRA.xml (hunk 1 above)
Comment (vickyjaiswal0308) [reply codesystems/CodeSystem-England-ConditionCodeRA.xml:37]: I will wait for @marcusfearnett-nhsdigital  to provide his thoughts on this and if there'll be any changes , we'll do the change . Once CodeSystems are finalized , I will do the changes in Examples too for the CodeSystem. After that , I think we'll be done with all the FHIR Assets.

Comment (ryma2fhir) [reply codesystems/CodeSystem-England-ConditionCodeRA.xml:37]: Marcus said on teams: "No change to display - they're already in use; they're agreed and aligned with Accessible Information standard terminology"

**Review by ryma2fhir — APPROVED** [review]

**Pull request merged**